actions/spring-generations/main.go

```diff
@@ -121,7 +121,9 @@ func main() {
 		panic(fmt.Errorf("unable to marshal content\n%w", err))
 	}
 
-	fmt.Printf("::set-output name=content::%s\n", strings.ReplaceAll(string(b), "\n", "%0A"))
+	actions.Outputs{
+		"content": strings.ReplaceAll(string(b), "\n", "%0A"),
+	}.Write()
```

Comment:
Same here. Is it still necessary to join on `%0A` here (urlencoded `\n`)? If we are handling multiline strings when we write them out?

Reply:
Same